frontend/src/components/CompanyInput/CompanyInput.js

```diff
@@ -16,8 +16,12 @@ export default class CompanyInput extends Component {
         this.state = {
             companyName: ""
         }
+        this.handleClick = this.handleClick.bind(this)
     }
 
+    // route the user to the information component with the company name they input
+    handleClick (){ this.props.history.push("/information/"+this.state.companyName);}
```

Comment:
Change `handleClick` to `handleSubmit` after adding `onSubmit` attribute to `<Form />`.